nibabel/analyze.py

```diff
@@ -892,6 +892,17 @@ def _chk_pixdims(hdr, fix=False):
             rep.fix_msg = ' and '.join(fmsgs)
         return hdr, rep
 
+    @classmethod
+    def is_header(klass, binaryblock):
+        if len(binaryblock) < klass.sizeof_hdr:
+            raise ValueError('Must pass a binary block >= %d bytes' %
+                             klass.sizeof_hdr)
+
+        hdr = np.ndarray(shape=(), dtype=header_dtype,
```

Comment:
Rename `hdr` to `hdr_struct` to indicate it's not a header object at this point?